src/installer/corehost/cli/apphost/static/CMakeLists.txt

```diff
@@ -56,8 +56,91 @@ if (CLR_CMAKE_TARGET_WIN32 AND (CLR_CMAKE_TARGET_ARCH_ARM OR CLR_CMAKE_TARGET_AR
     target_link_libraries(singlefilehost Advapi32.lib shell32.lib)
 endif()
 
+# Path like: artifacts/bin/coreclr/Windows_NT.x64.Release/lib  or
+#            /root/runtime/artifacts/transport/coreclr/lib
+set(CORECLR_STATIC_LIB_LOCATION "$ENV{__CoreClrArtifacts}lib")
```

Comment:
A nit - can we have the env var without the trailing slash for the sake of consistency with cmake variables for paths (e.g. the CMAKE_CURRENT_SOURCE_DIR)?

Reply:
It looks like I am not adding this trailing slash. It gets here all the way from msbuild `CoreCLRArtifactsPath` 

I would rather keep it as is. I mean - rather than detecting and trimming the trailing slash, if present, before exporting in .cmd/.sh

We probably have the same thing for other variables that come from msbuild and just add trailing slashes on concat.   
I assume duplicate path separators are ignored on Unix as well as on Windows.